subworkflows/local/standardisation_profiles.nf

```diff
@@ -156,15 +151,7 @@ workflow STANDARDISATION_PROFILES {
 
     MOTUS_MERGE ( ch_profiles_for_motus, ch_input_databases.motus.map{it[1]}, motu_version )
 
-    if ( params.generate_biom_output ) {
-        ch_standardised_tables = ch_standardised_tables.mix ( MOTUS_MERGE.out.biom )
-    } else {
-        ch_standardised_tables = ch_standardised_tables.mix ( MOTUS_MERGE.out.txt )
-    }
-    ch_versions = ch_versions.mix( MOTUS_MERGE.out.versions )
```

Comment:
This should be kept! Sorry, Don't know why this is not below the MOTUS_MERGE call itself

Reply:
My bad!

Reply:
My bad!